### PR TITLE
WIP: New Vision and Mission Statement

### PR DESCRIPTION
**Pull request by dasanchez**
### _includes/vision.md
Updated with the text from the [Mission Review](https://github.com/tomeshnet/documents/blob/master/tomesh_mission/2018/2018_mission_review.md) document.

### about.md
Replaced **Our Efforts** with a line linking to the Projects page.

### index.html
Replaced _We are building a mesh network in Toronto_ with the new mission statement:

_We are helping communities create better networks that rely on open source and peer-to-peer technologies, while promoting digital literacy and personal privacy_.

At first it seemed too wordy, but it has grown on me 😄 

### get-involved.md
Added a link to the Projects page.

### CODE REVIEW

**Review by makew0rld — APPROVED** [review]

**Comment by Shrinks99** [comment]
How does the mission statement look on the homepage?  It's a fair bit longer than the old one and I'm wondering if it needs typography fixes.

Screenshot?

**Comment by dasanchez** [comment]
@Shrinks99 here you go:

tomesh-home

**Comment by Shrinks99** [comment]
It's become big!  I would try to adjust it myself but I can't run the Jekyll bundler.  Thoughts @garrying?

**Comment by dasanchez** [comment]
This is what it looks like after replacing the `<h2>` with `<h4 style="font-weight: normal;">`:

tomesh-home-2

**Comment by ansuz** [comment]
> We are helping communities create better networks that rely on open source and peer-to-peer technologies, while promoting digital literacy and personal privacy.

What about: 

> We help communities create better networks with open source and peer-to-peer tech, while promoting digital literacy and personal privacy.

It's a tiny bit shorter.

**Comment by Shrinks99** [comment]
@ansuz I like shortening it on the homepage as much as possible, it's good to be concise!  I think we could probably get away with removing "personal" from "personal privacy" as that is something we can probably rely on other people to infer.  

I don't know if I like changing "technologies" to "tech" as it changes the tone a bit?  Perhaps "systems" or "tools" would fit better?

**Comment by dasanchez** [comment]
I also prefer "technologies" over "tech":

**We help communities create better networks with open source and peer-to-peer technologies, while promoting digital literacy and privacy.**

**Comment by Shrinks99** [comment]
New screenshot with the shortened title?  (I'm off school now!  I'll fix Jekyll at some point).

**Comment by dasanchez** [comment]
@Shrinks99 the tagline is back at `<h2>` now:

tomesh-home-3

**Comment by Shrinks99** [comment]
I'd be pleased with adding a class and scaling it down so it fits on two lines.  Very sorry I cannot do this myself due to stupid Ruby problems.

**Comment by dasanchez** [comment]
@Shrinks99 the tagline css is under `/_sass/_layout.scss` -> `.feature { h2...`

I feel like posting screenshot after screenshot is somewhat inefficient, so I threw the `_site` contents from the latest commit on dat (I'll keep it live until you've had a chance to copy it):

```dat://00b887569e44b83dde9a4fb89bd19e8fc77c8df03a79d21d3fdaeb74dda38f50/```

IPFS is broken on my node 😅

**Comment by dcwalk** [comment]
Looking good, I think there might be an ambiguity introduced by the dependent clause? 

"We help communities create better networks with open source and peer-to-peer technologies, *while promoting digital literacy and privacy*."

What is the thing that the conjunction "while" is referring to?
- the act of helping communities?
- the open source/p2p tech?

Not sure the intent so don't know how to remedy it, I'd say something like...

"We help communities create better networks with open source and peer-to-peer technologies, **in order to promote** digital literacy and privacy."
or
"We help communities create better networks with open source and peer-to-peer technologies **that promote** digital literacy and privacy."

could make clear the emphasis (helping, tech)

**Comment by Shrinks99** [comment]
@dcwalk  I like "that promote" better for the reasons you've described and also that it combines both descriptors of our goals into a singular larger action.  "That promote" gets a 👍 from me!

**Review by Shrinks99 — APPROVED** [review]
Looks good!
